KaggleTorrent/main.py

```diff
@@ -57,13 +63,47 @@ def main():
     db_engine = db_connection_handler.create_sqlalchemy_engine()
     print("Connected to the database")
 
-    if command == 'generate':
-
+    # CHECK USER VARIABLES
+    proceed = None
+
+    # Check db emptiness
+    if database_exists(db_engine.url):
+        if command == 'init':
+            print(f'Database {config.db_name} already exist. Provide database name that does not exist.',
```

Comment:
**Typo**: "Database {config.db_name} already exist**s**."

I'd suggest:
"Database {config.db_name} already exists. Please, provide a name that is not already in use for the KGTorrent database."